File: picture_library/src/main/java/com/luck/picture/lib/app/IApp.java
package com.luck.picture.lib.app;

import android.content.Context;

import com.luck.picture.lib.engine.PictureSelectorEngine;

/**
 * @author：luck
 * @date：2019-12-03 15:14
 * @describe：IApp
 */
public interface IApp {
    /**
     * 获取全局应用Application
     *
     * @return
     */
    Context getAppContext();

    PictureSelectorEngine getPictureSelectorEngine();
}

File: selector/src/main/java/com/luck/picture/lib/app/IApp.kt
package com.luck.picture.lib.app

import android.content.Context

/**
 * @author：luck
 * @date：2019-12-03 15:14
 * @describe：IApp
 */
interface IApp {
    fun getAppContext(): Context?
    fun getSelectorEngine(): SelectorEngine?
}